packages/driver/cypress/e2e/e2e/origin/cookie_behavior.cy.ts

```diff
@@ -0,0 +1,1454 @@
+describe('Cookie Behavior with experimentalSessionAndOrigin=true', () => {
```

Comment:
We should skip any tests we currently expect to fail.

Reply:
I think there is value for keeping the current tests with expected/actual assertions as we know if we change the current behavior, but none of them should currently fail (looks like I need to fix a few tests 😅 ).

Reply:
@chrisbreiding only test that I think should be skipped is the one in [5213499](https://github.com/cypress-io/cypress/pull/23438/commits/5213499b8b1e05cb376af553e7986c1a3489df5f), since the extension can set `SameSite=None` with `Secure=false` in firefox.

Reply:
My bad, I thought the failing tests were supposed to fail and related to the FIXME comments. So are those FIXME tests currently verifying incorrect behavior?

Reply:
@chrisbreiding correct, with corrective assertions commented out.